### ISSUE TITLE
Use Maven (or gradle, etc) to manage project dependencies

### ISSUE THREAD

**Issue by ms5984** [Enzias/EasyDuels#12]
I would like load this project locally; without maven's `pom.xml` I cannot tell for sure what dependencies the project is built against. If this is not possible, may you provide such information manually?

**Comment by Enzias** [comment]
Hello you can find all dependencies of the plugin in the plugin.yml file ^^

**Comment by ms5984** [comment]
Fair enough. To clarify, Maven manages dependencies required for building the project. So perhaps you've the spigot-api and VaultAPI jars as well as their versions. Otherwise I can't necessarily build your project.

**Comment by Enzias** [comment]
Ohh okay ^^' As you can see I don't use these dependencies manager and I don't know how to use them so for the moment that won't be possible but I'll try this later. Therefore, if you need to download the plugin you can find it on the spigot page and that will support me a lot ^^ Do not hesitate to send me a message on Discord if you have any issue though ;)

**Comment by ms5984** [comment]
Noted. if you could leave this issue open for me till then, that's fine. I'll test out the release, thanks!
